server/rating.js

```diff
@@ -0,0 +1,49 @@
+import express from "express";
+import fetch from "node-fetch";
+
+export const displayRating = express.Router();
```

Comment:
This line creates an express _router_, but the name of the variable that stores that new router is called `displayRating`, which is not clearly communicating to me what it is. It even sounds a little like an action rather than a thing, which makes me expect it to be a function when I see it mentioned elsewhere.